test/test.py

```diff
@@ -763,6 +764,29 @@ def test_name_substitution(self):
             with_slash_after = set(f1 for f1 in files_after if '/' in f1)
             self.assertEqual(expected_slash_after, with_slash_after)
 
+    def test_project_accessions(self):
+        uuid = 'eca05046-3dad-4e45-b86c-8720f33a5dde'
```

Comment:
There already is a test for accessions. It's using the same bundle. Please consolidate.